### ISSUE TITLE
Poor accuracy on HRSC2016 dataset

### ISSUE THREAD

**Issue by TerrafYassin** [Shank2358/GGHL#18]
Hi sir,
When execute your model on HRSC2016 on data you given to me i got the following results  : 
![image](https://user-images.githubusercontent.com/37477283/182112999-521f8934-4e92-4e9d-b521-1ed26511a322.png)

Need i to crop sub images from original images with an overlap like DOTA dataset or there is another problem ?

**Comment by Shank2358** [comment]
> 

Hi, Did you use the pre-trained weights? What is the epoch?

**Comment by Shank2358** [comment]
> Hi sir, When execute your model on HRSC2016 on data you given to me i got the following results : ![image](https://user-images.githubusercontent.com/37477283/182112999-521f8934-4e92-4e9d-b521-1ed26511a322.png)
> 
> Need i to crop sub images from original images with an overlap like DOTA dataset or there is another problem ?

Hi, Did you use the pre-trained weights? What is the epoch?

**Comment by TerrafYassin** [comment]
Yes sir  , i used darknet53.weights and the training epochs stopped on epoch 130 , after tested the best model on Test dataset i got 0.54 as mAP.

**Comment by Shank2358** [comment]
> Yes sir , i used darknet53.weights and the training epochs stopped on epoch 130 , after tested the best model on Test dataset i got 0.54 as mAP.

Hi, I'm re-experimenting this dataset with the settings you gave. I will tell you the result.

**Comment by Shank2358** [comment]
> Yes sir , i used darknet53.weights and the training epochs stopped on epoch 130 , after tested the best model on Test dataset i got 0.54 as mAP.

Hi, these are my results, I have uploaded the weight to google drive.
https://drive.google.com/file/d/1-D3C7Ul07frv8Uz-wt_gT7uQOJEtoTTm/view?usp=sharing
I only trained it for 50 epochs to achieve mAP=95. I re-updated the code. Maybe you need to check that the converted data labels are correct.
![image](https://user-images.githubusercontent.com/33946139/189475608-fdbf9d2f-6191-4413-a21e-3e632379c98d.png)
![image](https://user-images.githubusercontent.com/33946139/189475620-67399313-a7bf-4aad-83a1-621deedc6fd7.png)
![image](https://user-images.githubusercontent.com/33946139/189475623-6547e1dc-b9b3-4086-9d53-70064627d45f.png)
 This is the training log.
[log.txt](https://github.com/Shank2358/GGHL/files/9539847/log.txt)

I will reply to your other questions in the next few days.
